frontend/zulu/src/components/Login/register.component.js

```diff
@@ -26,7 +27,19 @@ const email = value => {
   }
 };
 
+
 const vusername = value => {
+  if (value.length < 3 || value.length > 20 || !isAlpha(value)) {
```

Comment:
How about the full name of [Hubert_Blaine_Wolfeschlegelsteinhausenbergerdorff_Sr.](https://en.wikipedia.org/wiki/Hubert_Blaine_Wolfeschlegelsteinhausenbergerdorff_Sr.)? :smile: 

Just kidding, guess 20 chars is OK :)